{"name": "test_execute_invalid_purchase3", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\t(No symbol) [0x00BFDCE3]\n\t(No symbol) [0x00B939D1]\n\t(No symbol) [0x00AA4DA8]\n\t(No symbol) [0x00AD019F]\n\t(No symbol) [0x00AD03AB]\n\t(No symbol) [0x00AFEE62]\n\t(No symbol) [0x00AEAF14]\n\t(No symbol) [0x00AFD57C]\n\t(No symbol) [0x00AEACC6]\n\t(No symbol) [0x00AC6F68]\n\t(No symbol) [0x00AC80CD]\n\tGetHandleVerifier [0x00E73832+2506274]\n\tGetHandleVerifier [0x00EA9794+2727300]\n\tGetHandleVerifier [0x00EAE36C+2746716]\n\tGetHandleVerifier [0x00CA6690+617600]\n\t(No symbol) [0x00B9C712]\n\t(No symbol) [0x00BA1FF8]\n\t(No symbol) [0x00BA20DB]\n\t(No symbol) [0x00BAC63B]\n\tBaseThreadInitThunk [0x76327D69+25]\n\tRtlInitializeExceptionChain [0x76FFB74B+107]\n\tRtlClearBits [0x76FFB6CF+191]", "trace": "self = <tests.chromeTests.testCheckoutProcess.TestCheckoutProcess object at 0x00000179E7021E40>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"463dfb666c8597f757883ff133fb9985\")>\n\n    @allure.description(\"the test is verify, that user can't execute a purchase with invalid finTrado data \")\n    def test_execute_invalid_purchase3(self, driver):\n        self.sign_in_modal.sign_in()\n        self.product_page.add_1item()\n        self.cart_bar.check_out_btn_click()\n        assert driver.current_url == \"https://qa.trado.co.il/checkout\"\n        assert self.common_actions.get_attribute_by_class(deli_dtls_lcs[\"deliv_details_status\"]) \\\n               == \"orderTimeline_time orderTimeline_current orderTimeline_current \"\n        assert self.common_actions.get_attribute_by_value(deli_dtls_lcs[\"bn_num_fld\"]) == \"987654321\"\n        self.delivery_details_page.set_invoice_details(\"Pizza Jon\", \"987654321\", \"jhongrey89@mail.com\", \"Paris\",\n                                                       \"Baget\", \"3\", \"a\", \"2\")\n        self.delivery_details_page.complete_purchase_btn_click()\n        self.payments_methods_page.fin_trado_click()\n>       assert self.common_actions.get_attribute_by_class(\n            payments_mthds_lcs[\"trado_modal_open\"]) == \"MuiDialog-container MuiDialog-scrollPaper css-ekeie0\"\n\ntests\\chromeTests\\testCheckoutProcess.py:372: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nsrc\\utils\\commonActions.py:34: in get_attribute_by_class\n    attribute = wdw(self.driver, 5).until(ec.visibility_of_element_located(locator)).get_attribute(\"class\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"463dfb666c8597f757883ff133fb9985\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x00000179E6E28CA0>, message = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       Backtrace:\nE       \t(No symbol) [0x00BFDCE3]\nE       \t(No symbol) [0x00B939D1]\nE       \t(No symbol) [0x00AA4DA8]\nE       \t(No symbol) [0x00AD019F]\nE       \t(No symbol) [0x00AD03AB]\nE       \t(No symbol) [0x00AFEE62]\nE       \t(No symbol) [0x00AEAF14]\nE       \t(No symbol) [0x00AFD57C]\nE       \t(No symbol) [0x00AEACC6]\nE       \t(No symbol) [0x00AC6F68]\nE       \t(No symbol) [0x00AC80CD]\nE       \tGetHandleVerifier [0x00E73832+2506274]\nE       \tGetHandleVerifier [0x00EA9794+2727300]\nE       \tGetHandleVerifier [0x00EAE36C+2746716]\nE       \tGetHandleVerifier [0x00CA6690+617600]\nE       \t(No symbol) [0x00B9C712]\nE       \t(No symbol) [0x00BA1FF8]\nE       \t(No symbol) [0x00BA20DB]\nE       \t(No symbol) [0x00BAC63B]\nE       \tBaseThreadInitThunk [0x76327D69+25]\nE       \tRtlInitializeExceptionChain [0x76FFB74B+107]\nE       \tRtlClearBits [0x76FFB6CF+191]\n\nC:\\Users\\artem\\PycharmProjects\\POMTrado\\venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:95: TimeoutException"}, "description": "the test is verify, that user can't execute a purchase with invalid finTrado data ", "start": 1680369119263, "stop": 1680369129161, "uuid": "d49479ef-b256-4d95-b7fe-3fae72f0709d", "historyId": "9e849dc9da7887cc7d6833a87c16a51e", "testCaseId": "0d45805f2dfc493e1cbd6d6ee6ee514d", "fullName": "tests.chromeTests.testCheckoutProcess.TestCheckoutProcess#test_execute_invalid_purchase3", "labels": [{"name": "parentSuite", "value": "tests.chromeTests"}, {"name": "suite", "value": "testCheckoutProcess"}, {"name": "subSuite", "value": "TestCheckoutProcess"}, {"name": "host", "value": "krot"}, {"name": "thread", "value": "8896-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.chromeTests.testCheckoutProcess"}]}